project/WartRemoverSettings.scala

```diff
@@ -0,0 +1,42 @@
+import sbt.Compile
+import sbt.Keys.compile
+import wartremover.{Wart, wartremoverErrors, wartremoverWarnings}
+
+
+object  WartRemoverSettings {
+
+  lazy val wartRemoverWarning = {
+    val warningWarts = Seq(
+      Wart.JavaSerializable,
+      Wart.StringPlusAny,
```

Comment:
Could you please move this to wartErrors. It will prevent from:

```
case class Vrn(value: String)

val vrn = Vrn(...)

val url =  s"${baseUrl(...)}/path/path/${vrn}"  


```

Reply:
fixed!